blockchain/contracts/HelloWorld.sol

```diff
@@ -29,16 +37,20 @@ contract HelloWorld {
         nftOwner = payable(0xC77787e364E3420c0609a249F18De47430900f0C);
     }
 
-    function createDrop(uint256 _dropTime) public {
-        uint256 i;
-        // I choose "3" as a value for testing reasons
-        for (i = 0; i < 3; i++) {
+    function createDrop(uint256 _dropTime, uint256 _numberOfNFTS) public {
+        for (uint256 i = 0; i < _numberOfNFTS; i++) {
             mockNFT(_dropTime, i + 1);
         }
+        // We need to let users buy 1 NFT instead of 5% if there are less than 20
+        if (_numberOfNFTS < 20) {
+            fivePercent = 1;
+        } else {
+            fivePercent = (_numberOfNFTS * 5) / 100;
+        }
```

Comment:
Is there a way to provide a name which is more independent from the hardcoded implementation?
Like
```
purchaseLimit = (_numberOfNFTs * percentLimit) / 100;
nftPurchaseLimit = max(1, purchaseLimit);
```

I don't know however, if such a `max` function exists (for convenience).

Reply:
Andre decided it should be 5%. If you would like to change that please talk to him during the next meeting :) I don't see a reason why we should force the drop creator to fill out an additional field if it's not necessary.

Reply:
Or were you only talking about the name itself?

Reply:
Yes, I was talking about the software-engineering part of code like potentially being able to change code easily and the variable name, no new requirement or functionality part.

Reply:
Will change the names.

Reply:
Also for the bottom part, how difficult or easy is it in Solidity to use a more generic variable without hardcoded integer symbols? For example in the error string below.

Reply:
What do you mean by "error string below"?

Reply:
In line 76